### ISSUE TITLE
Make the software updater look for other commands

### ISSUE THREAD

**Issue by anupam-git** [anupam-git/nx-software-updater#4]
_From @UriHerrera on December 29, 2017 21:21_

The software updater looks for the command `apt` for its functionality however we're not using `apt` in Nitrux which means it doesn't work.

We're using `pacman` commands to install and remove packages through a modified version of [pacapt](https://github.com/icy/pacapt). Pacapt also relies on `apt` but our modified version uses a command called `nxi` which is a symlink to `apt` (so all `apt` commands work the same but instead of using `apt update` you use `nxi update`).

In other words, the software updater would have to look for `nxi` if it can't find `apt`.

_Copied from original issue: nomad-desktop/nx-software-updater#2_

**Comment by anupam-git** [comment]
Updated code now supports `nxi`.
